dev-master/zen/templates/validator.py

```
@@ -0,0 +1,155 @@
+        missing = [field for field in required_fields if not entry.get(field)]
+        if missing:
+            report.add("error", f"Missing required registry fields: {', '.join(missing)}")
+
+        if not entry.get("tags"):
+            report.add("warning", "Template has no tags; consider adding for better discovery.")
+
+    def _validate_render(self, entry: Dict[str, Any], variables: Dict[str, Any], report: ValidationReport) -> None:
+        """Attempt to render template with provided variables."""
+        try:
+            self.engine.render_by_id(entry["id"], variables)
+        except Exception as exc:  # pylint: disable=broad-except
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)